content/en/dashboards/guide/related_links.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Make dashboards widgets an entry point of your workflows with related links
+kind: guide
+aliases:
+  - /logs/guide/custom-links
+  - /logs/guide/custom_links
```

Comment:
Since this page is completely new (and does not exist already on the docs site), these aliases will not work. 

```suggestion
aliases:
```